codes/csharp/test/chapter_array_and_linkedlist/ArrayTest.cs

```diff
@@ -0,0 +1,70 @@
+﻿// File: ArrayTest.cs
+// Created Time: 2022-12-16
+// Author: mingXta (1195669834@qq.com)
+
+using NUnit.Framework;
```

Comment:
Could we merge `ArrayTest.cs` and `Array.cs` . Compared to the engineering correctness, the simplicity is more significant for this book.

Reply:
Sure, I'll modify it.